resources/responses.js

```diff
@@ -279,14 +279,28 @@ let Responses = {
     return obj;
   },
   stack: (sev) => {
+    // for stack marker
```

Comment:
Do you think it'd be worth renaming this to `stackMarker` and `stackMarkerOn` for clarity?